packages/editor/src/components/widgets/table/row/SelectRow.css

```diff
@@ -1,14 +1,17 @@
 .selectable-row {
   cursor: pointer;
 }
-.selectable-row:where(:hover) {
+.selectable-row:where(:hover),
+.selectable-row:where(:hover) .input {
```

Comment:
where is not needed if there is only one thing inside, just do `.selectable-row:hover`